### ISSUE TITLE
uninitialized constant Formulary::HOMEBREW_CORE_FORMULA_REGEX

### ISSUE THREAD

**Issue by ejamroz** [Homebrew/homebrew#42977]
The above error is thrown when trying to run: 
```
brew update
```

This is the output:
```
/usr/local/Library/Homebrew/formulary.rb:226:in `loader_for'
/usr/local/Library/Homebrew/formulary.rb:176:in `factory'
/usr/local/Library/Homebrew/cmd/update.rb:173:in `block in report'
/usr/local/Library/Homebrew/cmd/update.rb:159:in `each_line'
/usr/local/Library/Homebrew/cmd/update.rb:159:in `report'
/usr/local/Library/Homebrew/cmd/update.rb:24:in `update'
/usr/local/Library/brew.rb:140:in `<main>'
```
Link to related gist:
https://gist.github.com/ejamroz/4343d1098036115a9bc9

**Comment by DomT4** [comment]
See #42553. It's harmless, `brew update` again and it won't reproduce.

**Comment by ejamroz** [comment]
Thank you

On Sat, Aug 15, 2015 at 2:23 PM, Dominyk Tiller <notifications@github.com>
wrote:

> Closed #42977 <https://github.com/Homebrew/homebrew/issues/42977>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/issues/42977#event-383374120>.
>



-- 
University of Puget Sound
Computer Science and Mathematics '13
